src/website/app/App.js

```diff
@@ -73,13 +74,20 @@ export default function App({ className, demos }: Props) {
 
   return (
     <div>
-      <Root className={className}>
-        <Nav demos={siteDemos} />
-        <Main>
-          <Router demos={siteDemos} />
-          <Footer />
-        </Main>
-      </Root>
+      <Route
+        render={route => {
+          const isChromeless = route.location.search === '?chromeless';
+          return isChromeless
+            ? <Router demos={siteDemos} />
```

Comment:
Would it be possible to add a wrapper with some padding when in chromeless mode?  It is otherwise unbearable to use with stuff butting right up agains the viewport.

Reply:
Yes, it would be trivial to add such a wrapper. However, that wrapper would then apply to the individual example chromeless pages too, without some _non-trivial_ routing/rendering logic. Those example pages already have [a small amount of padding added to their LivePreview](https://github.com/mineral-ui/mineral-ui/pull/371/files#diff-85abe98e739c3b9285bf8adcb93201ceR20); adding yet more padding would result in too much.

Chromeless mode is only intended for the individual example case. It happens to work on other pages because of its implementation details, but there is little-to-no value in optimizing those cases, imo.